tools/rewriter/SourceRewriter.cpp

```diff
@@ -513,16 +513,14 @@ class FnPtrCall : public RefactoringCallback {
 
     std::string new_expr =
         "IA2_CALL("s + old_callee.str() + ", " + mangled_ty;
-    
+
```

Comment:
It'd be nice to merge in #518 first so we don't have these same formatting changes in a bunch of different PRs (I've seen this one, too, in a bunch of mine).